pkg/cmd/agent/stop.go

```diff
@@ -25,24 +28,46 @@ func NewCmdAgentStop(f *factory.Factory) *cobra.Command {
 			If the agent is already stopped the command returns an error.
 		`),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// create a bubbletea program to manage the output of this command
-			l := io.NewPendingCommand(func() tea.Msg {
-				org, agent := parseAgentArg(args[0], f.Config)
-				_, err := f.RestAPIClient.Agents.Stop(org, agent, force)
-				if err != nil {
-					return err
+			switch agents := len(args); {
+			case agents == 0:
+				// No agents slug/UUID passed in, return an error
+				return errors.New("Please specify at least one agent to stop.")
+			case agents == 1:
+				// Stop the given agent from args[0]
+				return stopAgent(args[0], f, force)
+			case agents >= 2:
+				// Construct an agentStopErrors variable to construct errors
+				var agentStopErrors error
+				for _, agent := range args {
+					err := stopAgent(agent, f, force)
+					// Append to agentStopErrors if there was an error stopping an agent
+					if err != nil {
+						agentStopErrors = errors.Join(agentStopErrors, err)
+					}
```

Comment:
Couldn't his all be handled with `>= 1`? The length of `args` would just be `1` and then it would run the loop once?